docker/1.3.0/final/Dockerfile.cpu

```diff
@@ -48,6 +48,9 @@ RUN framework_installable_local=$(basename $framework_installable) && \
 RUN pip install onnx==1.2.1 \
                 keras-mxnet==2.2.2
 
+RUN mkdir /root/.keras && \
+    echo '{"image_data_format": "channels_first"}' > /root/.keras/keras.json
```

Comment:
This looks like a good reference to add here - https://github.com/awslabs/keras-apache-mxnet/blob/master/docs/mxnet_backend/performance_guide.md#channels-first-image-data-format-for-cnn

Reply:
done